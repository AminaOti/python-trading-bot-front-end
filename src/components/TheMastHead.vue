<template>
  <header class="header-bar">
    <div class="hello-there"></div>
    <H2> Thicc Dough Crypto</H2>
  </header>
</template>

<script>
import "../assets/css/theme-colors.scss";

export default {
  name: "TheMasthead",
};
</script>

<style scoped>
.header-bar {
  color: var(--secondary-color);
  background-color: var(--primary-color);
  padding: 1%;
}
h2 {
  padding-left: 7%;
}
</style>
