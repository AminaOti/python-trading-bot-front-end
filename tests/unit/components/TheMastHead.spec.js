import { shallowMount } from "@vue/test-utils";
import TheMastHead from "@/components/TheMastHead.vue";

describe("The header - TheMastHead.vue", () => {
  it("should render the title passed through the props", () => {
    const siteTitle = "Mock Site Title";
    const wrapper = shallowMount(TheMastHead, {
      props: { siteTitle },
    });
    expect(wrapper.text()).toMatch(siteTitle);
  });
  it("should match snapshot", () => {
    const siteTitle = "Mock Site Title";
    const wrapper = shallowMount(TheMastHead, {
      props: { siteTitle },
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
