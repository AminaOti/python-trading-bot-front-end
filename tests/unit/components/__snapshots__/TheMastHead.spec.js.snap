// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The header - TheMastHead.vue should match snapshot 1`] = `
<header
  class="header-bar"
>
  <div
    class="hello-there"
  />
  <h2>
    Mock Site Title
  </h2>
</header>
`;
